include/klee/Interpreter.h

```diff
@@ -102,8 +103,8 @@ class Interpreter {
   ///
   /// \return The final module after it has been optimized, checks
   /// inserted, and modified for interpretation.
-  virtual const llvm::Module * 
-  setModule(llvm::Module *module, 
+  virtual llvm::Module *
+  setModule(std::vector<std::unique_ptr<llvm::Module> > &module,
```

Comment:
I would suggest changing `&module` to `&modules` here and adding a `@param` explaining the purpose of having a vector here (similar to klee::linkModules)?